### PR TITLE
"Fix" for Issue #308

### PR DESCRIPTION
**Pull request by DerDevHD**
Dear @EndermanLetsPlay

**This is my result for your Issue #308 
I Hope you are happy with this, if not, please suggest under this pull request.**

Infos for Jason:
- Added the /p transferownership Subcommand,
- Added in plugin.yml the permissions,
- Added the messages in eng.ini, deu.ini and langtemplate.ini

### CODE REVIEW

File: resources/langtemplate.ini [hunk 1]
```diff
@@ -227,4 +227,12 @@ clone.desc=
 clone.usage=/p clone <> []
 clone.wrongid=
 clone.nomoney=
-clone.success=
\ No newline at end of file
+clone.success=
+
+
```
Comment (Aericio) [inline resources/langtemplate.ini:232]: extra space

File: src/MyPlot/subcommand/TransferOwnershipSubCommand.php [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+namespace MyPlot\subcommand;
+
+
+use MyPlot\MyPlot;
+use MyPlot\Plot;
+use pocketmine\command\CommandSender;
+use pocketmine\utils\TextFormat;
+
+class TransferOwnershipSubCommand extends SubCommand {
+
```
Comment (Aericio) [inline src/MyPlot/subcommand/TransferOwnershipSubCommand.php:14]: coding inconsistency, should be:
```php
<?php
declare(strict_types=1);
namespace MyPlot\subcommand;

use imports;

class TransferOwnershipSubCommand extends SubCommand
{
//no eol
```

I notice that class definition formats aren't consistent throughout the project though, so @jasonwynn10 should probably choose one: `...{` or `...eol{`

File: src/MyPlot/subcommand/TransferOwnershipSubCommand.php [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+			return true;
+		}
+		foreach($plots as $plot) {
+			$plot->owner = $toUser;
+			MyPlot::getInstance()->savePlot($plot);
+		}
+		$sender->sendMessage(TextFormat::WHITE . $this->translateString("transferownership.success", [count($plots), $fromUser, $toUser]));
+		return true;
+	}
+
```
Comment (Aericio) [inline src/MyPlot/subcommand/TransferOwnershipSubCommand.php:47]: extra space

File: src/MyPlot/subcommand/TransferOwnershipSubCommand.php [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+	public function canUse(CommandSender $sender): bool {
+		return $sender->hasPermission("myplot.admin.transferownership");
+	}
+
+	/**
+	 * @param CommandSender $sender
+	 * @param array $args
+	 * @return bool
+	 */
+	public function execute(CommandSender $sender, array $args): bool {
```
Comment (Aericio) [inline src/MyPlot/subcommand/TransferOwnershipSubCommand.php:28]: coding inconsistency, should be:
```php
public function execute(CommandSender $sender, array $args) : bool {
```

File: src/MyPlot/subcommand/TransferOwnershipSubCommand.php [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+use pocketmine\command\CommandSender;
+use pocketmine\utils\TextFormat;
+
+class TransferOwnershipSubCommand extends SubCommand {
+
+	/**
+	 * @param CommandSender $sender
+	 * @return bool
+	 */
+	public function canUse(CommandSender $sender): bool {
```
Comment (Aericio) [inline src/MyPlot/subcommand/TransferOwnershipSubCommand.php:19]: coding inconsistency, should be:
```php
public function canUse(CommandSender $sender) : bool {
```

File: resources/langtemplate.ini (hunk 1 above)
Comment (Aericio) [inline resources/langtemplate.ini:232]: extra space

**Comment by Aericio** [comment]
`/p transferownership` is a pretty lengthy command... would `/p transfer` not suffice?

**Comment by DerDevHD** [comment]
Fixed everything requested...

**Comment by EndermanLetsPlay** [comment]
Hi there! Thank you very much for the update.
To what extent has the function been implemented in the plugin? Do I need to download anything?

**Comment by DerDevHD** [comment]
Hello,

You just have to wait, until Jason accept this pull request and merge it into his repository.
Then, all changes are implemented in the main plugin and you can use the latest dev build from poggit.

If it's taking too long, I could make it as an own project on poggit and build my version of the plugin.

Best regards,
DerDevHD.

**Comment by DerDevHD** [comment]
I dunno, kinda useles

**Pull request closed without merging**